src/web/app/pages-help/instructor-help-page/instructor-help-questions-section/instructor-help-questions-section.component.ts

```diff
@@ -90,6 +92,23 @@ export class InstructorHelpQuestionsSectionComponent extends InstructorHelpSecti
     showRecipientNameTo: [FeedbackVisibilityType.INSTRUCTORS, FeedbackVisibilityType.RECIPIENT],
   };
 
+  readonly numericalScaleResponses: Response<FeedbackNumericalScaleResponseDetails>[] = [
```

Comment:
- Add `example` prefix to the variable
- You may want to check the example in V6:
  
  <img width="1118" alt="Screen Shot 2020-06-03 at 12 23 18 AM" src="https://user-images.githubusercontent.com/7261051/83544544-a694bc00-a530-11ea-8b9b-42a7ff497148.png">

  Based on that:
  - You need to show the list of responses as well
  - Because of that, you need to add more data points. Although you can get statistics from one (or even zero) responses, it's better as a demo if we can show better. Particularly for this question type, since we have max/min/average, the example should be able to capture it as well.

Reply:
Updated!

Reply:
The example box in V6 shows the whole card, including the question description and the detailed responses. At the very least, you should show the detailed responses, otherwise user can't know how the statistics are calculated.